#DCモーターの角度指定
float32 hat_shoulder_angle
float32 sword_A_shoulder_angle
float32 sword_B_shoulder_angle

#DCモーターの位置指定
float32 launcher_linear_pos

#DCモーターの出力指定
float32 launcher_power
float32 omni1_power
float32 omni2_power
float32 omni3_power
float32 omni4_power

#サーボモーターの状態更新
float32 hat_finger_angle
float32 sword_A_finger_angle
float32 sword_B_finger_angle
float32 hat_wrist_angle
float32 sword_A_wrist_angle
float32 sword_B_wrist_angle